tests/integration/testdata/publish/main.py

```diff
@@ -0,0 +1,2 @@
+def lambda_handler(event, context):
```

Comment:
Is this necessary? Does the publish command validate and error if it doesn't find the actual file?

Reply:
I think CreateApplication looks for Resource in the template, and will error out if the CodeUri under lambda resource is not a valid S3 path.